src/Proj4.jl

```diff
@@ -1,5 +1,8 @@
 module Proj4
 
+using BinDeps
+@BinDeps.load_dependencies
+
 const libproj = "libproj"
```

Comment:
When I am better rested, or the packages are updated for the last bits of v0.4 deprecations, I will understand just what the options are for `load_dependencies`, as indeed I believe it will define a handle to the library. In my mind this was related to doing precompilation effectively which I didn't want to tangle this up with.